packages/salesforcedx-sobjects/test/integration/config/project-scratch-def.json

```diff
@@ -0,0 +1,8 @@
+{
+  "orgName": "Your Company",
+  "edition": "Developer",
+  "orgPreferences": {
+    "enabled": ["Translation"],
```

Comment:
Why did you choose to enabled the "Translation" preference, in particular?

Reply:
I just copied it from one of my existing scratch-def files. I can remove it.